### ISSUE TITLE
SImplifying bitflags

### ISSUE THREAD

**Issue by SonnyX** [sciter-sdk/rust-sciter#89]
Another potential way to improve code, albeit at the expense of adding another dependency
https://github.com/sciter-sdk/rust-sciter/compare/master...Alovchin91:feature/bitflags

**Comment by pravic** [comment]
But it's safer. I have to do this eventually.

**Comment by SonnyX** [comment]
Here is another repo implementing some more bitflags, along with mutability changes and other improvements:
https://github.com/sciter-sdk/rust-sciter/compare/master...kwdkk:master

**Comment by pravic** [comment]
Yeah. The bitflags is a total win but other changes seem controversial and arguable.
